### PR TITLE
Implement boardHelper

### PR DESCRIPTION
**Pull request by piotrkaminski95**

### CODE REVIEW

File: src/main/java/com/codecool/model/board/Board.java [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
 
+import com.codecool.controller.BoardController;
 import com.codecool.model.Directions;
+import com.codecool.model.Position;
 import com.codecool.model.creature.Herbivore;
 
 public class Board {
 
      private Cell[][] board;
+     private BoardController boardController;
```
Comment (kret0) [inline src/main/java/com/codecool/model/board/Board.java]: maybe rename BoardControler -> BoardHelper

File: src/main/java/com/codecool/model/creature/Creature.java [hunk 2]
```diff
@@ -5,4 +5,5 @@
 public interface Creature {
 
     Directions move();
+    int getSpeed();
```
Comment (kret0) [inline src/main/java/com/codecool/model/creature/Creature.java]: we dont use Speed anyways

File: src/main/java/com/codecool/controller/BoardController.java [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+            }
+            case SW: {
+                return indexSouthWestIndex(pos);
+            }
+            default:
+                throw new IllegalArgumentException();
+        }
+    }
+
+    private void handleBorder(Position pos) {
```
Comment (kret0) [inline src/main/java/com/codecool/controller/BoardController.java]: not used. And really important :D

**Review by kret0 — CHANGES_REQUESTED** [review]
.

**Comment by kret0** [comment]
seems more ok now :dancer:

**Review by kret0 — APPROVED** [review]
ok

**Comment by piotrkaminski95** [comment]
thx 4 review.

**Pull request closed without merging**